rocAL/include/pipeline/enum_registry.h

```diff
@@ -76,6 +85,22 @@ class EnumRegistry {
         return (it != _enum_map.end()) ? it->second : "";
     }
 
+    /*!
+     * \brief Convert integer value to enum and return as std::any
+     * \param enumTypeName The name of the enum type
+     * \param value The integer value to convert
+     * \return The converted enum value wrapped in std::any
+     * \throws std::runtime_error if enum type is not registered
+     */
+    std::any convertIntToEnum(const std::string& enumTypeName, int value) const {
+        auto it = _name_to_converter_map.find(enumTypeName);
+        if (it != _name_to_converter_map.end()) {
+            return it->second(value);  // Call the conversion function and return std::any
+        } else {
+            throw std::runtime_error("No conversion function registered for enum: " + enumTypeName);
```

Comment:
Blocking: `convertIntToEnum` throws `std::runtime_error`, but `<stdexcept>` is not included in this header. Several headers (e.g. `meta_data/meta_data_reader.h`) include `pipeline/enum_registry.h` without bringing in `<stdexcept>`, so this change breaks the build. Please add the include here.